SCClassLibrary/Common/Collections/EnvironmentRedirect.sc

```diff
@@ -31,11 +31,18 @@ EnvironmentRedirect {
 		dispatch.value(key, obj);
 	}
 
+	removeAt { arg key;
+		var hadKey = true;
+		var result = envir.removeAtFail(key) { hadKey = false; nil };
```

Comment:
I personally find the postfix notation here not as clear as it could/should be? Or are there some performance gains here in this case?

At least it took me some minutes in order to grasp what's going on here, which is essentially

```supercollider
removeAt {|key|
	var result = envir.removeAtFail(
		key: key,
		// function will only be evaluated
		// if the key is not present
		function: {
			// bail out early with nil
			^nil;
		},
	);
	dispatch.value(key, nil);
	^result;
}
```

The comments seem to be necessary b/c of the lacking docs of `removeAtFail` and due to its generic arg name.

Reply:
yes, that is good. I would still avoid keyword aruments in the class library if not necessary, because they are slower. Also here, they are redundant anyway. I'll add a fix.